retriever/lib/install.py

```diff
@@ -1,209 +0,0 @@
-from __future__ import print_function
-from __future__ import absolute_import
-import imp
-import os
-import sys
-from pkg_resources import parse_version
-from retriever.engines import choose_engine
-from retriever.lib.compile import compile_json
-from retriever.lib.defaults import DATA_DIR, SCRIPT_SEARCH_PATHS, VERSION
-from retriever.lib.tools import name_matches
-from retriever.lib.scripts import SCRIPT_LIST
-
-script_list = SCRIPT_LIST()
```

Comment:
If you print out this script_list in this module, it is empty.
print (script_list)

**_Follow the streps._**

- Install retriever using `pip install . -U` 
- Clean out the scripts using `retriever reset all`. 
- Clean up compiled scripts in working directory of retriever using git add -u (to add your changes) followed by git clean -fx (to clean all unwanted files)

- Then run a simple install 

```
from retriever import install_json
install_json('iris')
```